data/json/vehicle_part_locations.json

```diff
@@ -0,0 +1,120 @@
+[
+  {
+    "type": "vehicle_part_location",
+    "id": "structure",
+    "name": "Structure",
+    "z_order": 5,
+    "list_order": 1
+  },
+  {
+    "type": "vehicle_part_location",
+    "id": "armor",
+    "name": "Armor",
+    "//": "Shouldn't be displayed (the color is used, but not the symbol)",
+    "z_order": -2,
+    "list_order": 2
+  },
+  {
+    "type": "vehicle_part_location",
+    "id": "on_roof",
+    "name": "On Roof",
+    "z_order": 9,
+    "list_order": 3
+  },
+  {
+    "type": "vehicle_part_location",
+    "id": "roof",
+    "name": "Roof",
+    "//": "Shouldn't be displayed",
```

Comment:
Will this be displayed if it's the highest part when the PC is on a higher z-level looking down?

Reply:
No. This location is never rendered. Currently and still with this PR, when you're looking at a higher Z level the interior of some vehicles renders black and some is visible, but they are all visible if you lower your view to the vehicle's Z level, and the roof part itself is never visible afaik.